samples/subsys/mgmt/mcumgr/smp_svr/sample.yaml

```diff
@@ -94,3 +94,16 @@ tests:
       - mg100
     integration_platforms:
       - nrf52840dk_nrf52840
+  sample.mcumgr.smp_svr.bt.nrf54l15pdk.ext_flash:
+    extra_args:
+      - OVERLAY_CONFIG="overlay-bt.conf"
+      - DTC_OVERLAY_FILE="boards/nrf54l15pdk_ext_flash.overlay"
+      - mcuboot_CONF_FILE="boards/nrf54l15pdk_ext_flash.conf"
+      - mcuboot_EXTRA_DTC_OVERLAY_FILE="boards/nrf54l15pdk_ext_flash.overlay"
```

Comment:
you can skip this entries:
- rename boards/nrf54l15pdk_ext_flash.overlay -> boards/nrf54l15pdk_nrf54l15_cpuapp.overlay
- copy from sdk-mcuboot:boot/zephyr/boards/nrf54l15pdk_ext_flash.conf -> child_image/mcuboot/boards/nrf54l15pdk_nrf54l15_cpuapp.conf
- copy from sdk-mcuboot:boot/zephyr/boards/nrf54l15pdk_ext_flash.overlay -> child_image/mcuboot/boards/nrf54l15pdk_nrf54l15_cpuapp.overlay

then you don't need to also add this to west command.
See details here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/config_and_build/multi_image.html#permanent-configuration-changes-to-child-images

Reply:
@gchwier I don't want to have this as default nRF54l15_pdk configuration.